### ISSUE TITLE
Page 1054-operational-guide is quite out of date

### ISSUE THREAD

**Issue by PeterParslow** [agiorguk/gemini#107]
- Are the terms at 1.2.1 still in use?
- Are the documents referred to in 1.3 still useful?
- Update the contacts in 1.4
- Remove 5.1
- Possibly add some other 'solutions'

**Comment by archaeogeek** [comment]
I agree that it's extremely out of date now, but it feels as if updating it would require quite a lot of work and would be subject to change anyhow. I've done a quick search for this and it's only referred to in the following locations:

- https://github.com/agiorguk/gemini/blob/76dfbbd6ea0fdfdf53ff74716e9e973e5e459643/docs/1047-metadata-guidelines-for-geospatial-data-resources-part-3.asciidoc#L62
- https://github.com/agiorguk/gemini/blob/76dfbbd6ea0fdfdf53ff74716e9e973e5e459643/docs/1051-uk-gemini-v2-2-specification-for-discovery-metadata-for-geospatial-resources.asciidoc#L150

So we could argue that it could be quietly dropped! I think this is something that would be worth bringing up for discussion at the next meeting.

**Comment by archaeogeek** [comment]
Agree to remove the links from the two linked pages, and move the asciidoc file into an archive folder

**Comment by alexarobu** [comment]
This has been addressed by https://github.com/agiorguk/gemini/pull/199, which was merged into the dev branch
